lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -1897,10 +1897,11 @@ else if (lessonEntity.notPublished())
                                 boolean isAudio = mimeType.startsWith("audio/");
                                 UIComponent h5video = UIOutput.make(tableRow, (isAudio? "h5audio" : "h5video"));
                                 UIComponent h5source = UIOutput.make(tableRow, (isAudio? "h5asource" : "h5source"));
-                                if (lengthOk(height) && height.getOld().indexOf("%") < 0)
-                                h5video.decorate(new UIFreeAttributeDecorator("height", height.getOld()));
-                                if (lengthOk(width) && width.getOld().indexOf("%") < 0)
-                                h5video.decorate(new UIFreeAttributeDecorator("width", width.getOld()));
+				// HTML5 spec says % isn't legal in width, so have to use style
+                                if (lengthOk(height))
+                                h5video.decorate(new UIFreeAttributeDecorator("style", "width: " + height.getNew()));
```

Comment:
There was a request that when we add new code we not change indentation of old. Otherwise I wouldn’t have done it this way.

Lessons consistently does not use {} around single lines. I try to match the style of existing code when I edit.

> On Mar 29, 2016, at 3:44 PM, Sam Ottenhoff <notifications@github.com> wrote:
> 
> In lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java <https://github.com/sakaiproject/sakai/pull/2050#discussion_r57787452>:
> 
> > @@ -1897,10 +1897,11 @@ else if (lessonEntity.notPublished())
> >                                  boolean isAudio = mimeType.startsWith("audio/");
> >                                  UIComponent h5video = UIOutput.make(tableRow, (isAudio? "h5audio" : "h5video"));
> >                                  UIComponent h5source = UIOutput.make(tableRow, (isAudio? "h5asource" : "h5source"));
> > -                                if (lengthOk(height) && height.getOld().indexOf("%") < 0)
> > -                                h5video.decorate(new UIFreeAttributeDecorator("height", height.getOld()));
> > -                                if (lengthOk(width) && width.getOld().indexOf("%") < 0)
> > -                                h5video.decorate(new UIFreeAttributeDecorator("width", width.getOld()));
> > +				// HTML5 spec says % isn't legal in width, so have to use style
> > +                                if (lengthOk(height))
> > +                                h5video.decorate(new UIFreeAttributeDecorator("style", "width: " + height.getNew()));
> I know complaining about code style is annoying ..... but no braces and bad tabs/spaces makes this code really odd to read ....
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub <https://github.com/sakaiproject/sakai/pull/2050/files/3505fe17c0cabb8a583e2d7985fbf1b51446a759#r57787452>